mcs/class/System.Numerics/System.Numerics_xtest.dll.sources

```diff
@@ -0,0 +1,8 @@
+../../../external/corefx/src/Common/tests/System/PlatformDetection.cs
+
+# ../../../external/corefx/src/System.Runtime.Numerics/tests/*.cs
+../../../external/corefx/src/System.Runtime.Numerics/tests/BigInteger/*.cs
+
+../../../external/corefx/src/System.Numerics.Vectors/src/System/Numerics/ConstantHelper.cs
+
+../../../external/corefx/src/System.Numerics.Vectors/tests/*.cs
```

Comment:
please add a newline at the end (it confuses the dist script on linux otherwise).